### DISCUSSION TITLE
Just want to say thank you.

### DISCUSSION THREAD

**Discussion by nerlins** [Ross-Patterson/Portals-Desktop-Organization#51] [General]
I don't know where else to compliment you, so I am doing it here. This app is fantastic. This is now my DE FACTO replacement for Stardock Fences. I only found it through a Reddit search. More review sites need to list your app, because is is HANDS DOWN the best (if not better) alternative.

**Comment by Ross-Patterson** [comment]
Thanks a lot!

**Comment by pr3dd1** [comment]
Thank you very much. Portals is a great tool and I can see that you are passionate about it.

**Comment by PhoenixR1s1ng** [comment]
I could fill up a full page with how upset I am with Stardocks Fences 5, but if you notice their price tag for a permanent license or their over arching strategy to charge an annual subscription to not get locked out of it it is pretty self explanatory. But enough about them and the potential mass exodus that may arrive at your doorstep....

I am actually genuinely impressed with how good portals is looking. Honestly, if we are to look just at the folder portals alone, then Portals is the superior product in many ways. This is great because that is what I use the most. Now hopefully we can get word of mouth with what you have going here.

Thank you for all your hard work!

**Comment by Crowgasm** [comment]
Echoing this. I had so many problems with Fences that I felt like buying the Stardock support and forum people 4000lbs of Dulcolax. Great tool. Happily donated after testing it for a few weeks.

**Comment by Solonaut** [comment]
Just wanted to add my thanks as well. I was so happy to be able to kick Stardock Fences off my computer once and for all. An actual full replacement! Am extremely happy with Portals. What a relief. Bought the licence. Thank you!
